aws/resource_aws_elasticsearch_domain.go

```diff
@@ -13,6 +13,7 @@ import (
 	"github.com/hashicorp/errwrap"
 	"github.com/hashicorp/terraform/helper/resource"
 	"github.com/hashicorp/terraform/helper/schema"
+	"strings"
```

Comment:
Aha, didn't spot that `gofmt` and/or Goland's import optimiser doesn't do this for you. Seems odd that `goimports` seems to add newlines between stdlib stuff when adding it and then `gofmt` won't rectify that either which is annoying considering `gofmt` is so good otherwise.